packages/bau/bau.0.0.1/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "Leonid Rozenberg <leonidr@gmail.com>"
+authors: "Leonid Rozenberg <leonidr@gmail.com>"
+homepage: "https://github.com/rleonid/bau/"
+bug-reports: "https://github.com/rleonid/bau/issues"
+license: "Apache2"
+dev-repo: "https://github.com/rleonid/bau.git"
+build: [make]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "bau"]
+depends: [
+  "ocamlfind" {build}
+]
+available: [ocaml-version >= "4.02.3"]
```

Comment:
Does the version constraint need to be so strict, or could it be

`available: [ocaml-version >= "4.02.0"]`

?